api/v1beta1/tempest_webhook.go

```diff
@@ -84,8 +84,15 @@ func (r *Tempest) ValidateCreate() (admission.Warnings, error) {
 func (r *Tempest) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
 	tempestlog.Info("validate update", "name", r.Name)
 
-	// TODO(user): fill in your validation logic upon object update.
-	return nil, nil
+	oldTempest, ok := old.(*Tempest)
+	if !ok || oldTempest == nil {
+		return nil, errors.New("Unable to convert existing object.")
+	}
+
+	warnings := admission.Warnings{}
+	warnings = append(warnings, "You are updating an already existing instance of a Tempets CR! Be aware that changes won't be applied.")
```

Comment:
I'm sorry @kstrenkova for being inaccurate in the ticket description. We need to make this an error as we do not want to pass the update of the Tempest CR to the test-operator-controller.